pkg/pf/tests/testdata/TestDetailedDiffList/attribute_no_replace/added.golden

```diff
@@ -0,0 +1,35 @@
+tfbridgetests.testOutput{
```

Comment:
This is a very readable file, thank you. This will make for minimal head-scratching when we detect some changes. Very well done.